packages/editor/__tests__/model/fieldModel.spec.ts

```diff
@@ -70,4 +78,43 @@ describe('Field test', () => {
     field.update({ value: 'text' });
     expect(field.rawValue).toEqual({ data: { a: 2 }, value: 'text' });
   });
+
+  it('change component id', () => {
```

Comment:
Add more test cases to make sure that normal text will not be changed, like `'Please input here'`, `'{{ {input: 123} }}'`, `'{{ 'input' }}'` ,  `['input']`

Reply:
Done.